src/util/prop-definition.js

```diff
@@ -76,6 +85,8 @@ export default class PropDefinition {
           break;
       }
     });
+
+    this.attrTargetIsSource = this.attrTarget && (this.attrTarget === this.attrSource);
```

Comment:
Do we need to even have the first part of this check here (`this.attrTarget && `)?